apollo-router/src/lib.rs

```diff
@@ -579,9 +577,7 @@ impl FederatedServerHandle {
                     State::Startup => {
                         tracing::info!("starting Apollo Router")
                     }
-                    State::Running { address, .. } => {
-                        tracing::info!("listening on {} 🚀", address)
-                    }
+                    State::Running { .. } => {}
```

Comment:
yes It's intentional because it's now displayed there https://github.com/apollographql/router/pull/976/files#diff-32d4f76b267b5de3eb4c6dd0aefdc5cdfcf5e68b4b630b3ce7162c5dd1bae80eR176 with the configured endpoint

Reply:
sgtm!